### PR TITLE
Added iOS and Python Task Library Documentation

### PR DESCRIPTION
**Pull request by priankakariat**

### CODE REVIEW

**Review by khanhlvg — CHANGES_REQUESTED** [review]

File: tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md [hunk 1]
```diff
@@ -99,6 +99,140 @@ See the
...
+```
+
+Make sure the `.tflite` you will be using for inference is present in your app bundle.
+
+### Step 2: Using the model 
+
+#### Swift
+
+```swift
+// Imports
```
Comment (khanhlvg) [inline tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md:127]: I'd suggest reformat the code like this.
The do / catch makes the code a bit hard to read. Keyword `try` present there should be enough to let Swift users know that there's error thrown and they need to handle it.

```swift
// Initialization
guard let modelPath = Bundle.main.path(forResource: "mobilenetv3",
                                            ofType: "tflite") else { return }
let options = ImageClassifierOptions(modelPath: modelPath)

// Optional: Configure any additional options:
// options.classificationOptions.maxResults = 5

let classifier = try ImageClassifier.imageClassifier(options: options)

// Convert the input image to MLImage.
// There are other sources for MLImage. For more details, please see:
// https://developers.google.com/ml-kit/reference/ios/mlimage/api/reference/Classes/GMLImage
guard let image = UIImage (named: "burger.jpg"), let mlImage = MLImage(image: image) else { return }

// Run inference
let classificationResults = try classifier.classify(gmlImage: mlImage)
```

File: tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md [hunk 2]
```diff
@@ -99,6 +99,140 @@ See the
...
+
+```objc
+// Imports
+#import <TensorFlowLiteTaskVision/TFLTaskVision.h>
+
+// Initialization
+NSString *modelPath = [[NSBundle mainBundle] pathForResource:@"your_model_file_name"
+                                                      ofType:@"tflite"];
+
+TFLImageClassifierOptions *imageClassifierOptions =
```
Comment (khanhlvg) [inline tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md:174]: Please use shorter name like in the suggestions in the Swift example code to make the code a bit easier to read.
* imageClassifierOptions -> options
* imageClassifier -> classifier

File: tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md (hunk 1 above)
Comment (priankakariatyml) [reply tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md:127]: Changed the model name and image name based on the "Example Results" section for each task. For eg., image classifier illustrates results for sparrow.jpg on a bird classifier.

**Review by khanhlvg — APPROVED** [review]

**Comment by khanhlvg** [comment]
Merged in https://github.com/tensorflow/tensorflow/commit/7e76026b4d2b43d0f1f70a2a5a5af25d14d0add1

**Pull request closed without merging**